custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -232,3 +232,36 @@ outer_predict <- function(input_filename, target_type, model=NULL, positive_clas
     }
     predictions
 }
+
+predict_unstructured <- function(model=NULL, data, ...) {
+    .validate_unstructured_predictions <- function(to_validate) {
+        .is_instance_character_raw_null <- function(v) {
+            is.character(v) || is.raw(v) || is.null(v)
+        }
+
+        .is_instance_list_none <- function(v) {
+            is.list(v) || is.null(v)
+        }
+
+
+        if (.is_instance_character_raw_null(to_validate)) {
```

Comment:
this IF is empty

Reply:
yes, as we allow valuer to be None, str, bytes, tuple; it's more clean that to do:
```
if not Null, str, bytes tuple:
   error
else if tuple
else if str or bytes
....
etc
```

Reply:
***Fixed as you suggested***